dll/win32/winmm/midimap/midimap.c

```diff
@@ -234,11 +234,17 @@ static BOOL	MIDIMAP_LoadSettings(MIDIMAPDATA* mom)
 	if (ret == 2)
 	{
             static const WCHAR ci[] = {'C','u','r','r','e','n','t','I','n','s','t','r','u','m','e','n','t',0};
+            static const WCHAR sz[] = {'s','z','P','n','a','m','e',0};
```

Comment:
Since it's forked, can't we use normal strings in place of this crap? :P

Reply:
I'm just following the style. :grin:

Reply:
If it is forked, please make at least new stuff readable.

Reply:
Ok, fixed.